### ISSUE TITLE
What is the Intent String for Locus File picker?

### ISSUE THREAD

**Issue by Falcosc** [asamm/locus-api#19]
Hi,

would be nice to use a file picker in Locus design. This method comment told that there should be a locus file picker https://github.com/asamm/locus-api/blob/f7ec11aa807fafa35fad3fa64e131577eb3be91d/locus-api-android/src/main/java/locus/api/android/ActionFiles.kt#L92-L93 
but I did only find the location picker https://github.com/asamm/locus-api/blob/master/locus-api-android/src/main/java/locus/api/android/utils/LocusConst.java#L112

Or does this method use Locus file picker and the comment tells that it just does not work below API Version 231? I'm a little bit confused because in other methods you did use throw RequiredVersionMissingException instead of placing a comment.

Result handling looks very special, too. https://github.com/asamm/locus-api/blob/f2dcd5c181efd38e03f3358df2c34335b279f3da/locus-api-android-sample/src/main/java/com/asamm/locus/api/sample/MainActivity.kt#L241-L242
Requestcode 0 for file and 1 for directories. Nice to have this example.

**Comment by Falcosc** [comment]
Everything works fine.
I guess I did understand it now.
You did use org.openintents.action.PICK_DIRECTOR to give user possibility to select any preferred file selector.
And the request code is used to map response to requests and can be any number you like.

We don't need to throw an exception on api missmatch because user may have other file picker installed.

I would like to update your method doc next week to make it more clear.

**Comment by menion** [comment]
Good day Falco,
your second post is correct.
Check on Locus API version is not needed here, because this method may work even if Locus Map is not installed. Well weird to use it then, I know.

Anyway, as you wrote ... Locus Map has registered intent-filter for these two `org.openintents.action.PICK_FILE` , `org.openintents.action.PICK_DIRECTORY` keys. So request open Locus Map file picker and return result over intent.

Useful method in Android 4.x and less because you were able to also write into selected directory/fie. With Android 5+ it's usage is mainly for read-only access. Then it's maybe better to use native Android file picker over 
```
Intent intent = new Intent(Intent.ACTION_OPEN_DOCUMENT_TREE);
startActivityForResult(intent, REQUEST_CODE_OPEN_DIRECTORY);
```
Just suggestion ...

**Comment by Falcosc** [comment]
Good point, I did just want to ask if you would like to implement an Document style  picker because current file picker does only allow read access.

**Comment by menion** [comment]
Definitely not. I made this system in times, there was no picker for the files on the Android platform. This is no longer true, so if anyone needs access to files, he may use mentioned system `ACTION_OPEN_DOCUMENT_TREE` that is available since API 21  (which is since version 3.38+ also minimal Locus Map API version).

**Comment by Falcosc** [comment]
Thank you, should may be added to your Java doc.
